homeassistant/components/habitica/sensor.py

```diff
@@ -222,6 +222,7 @@ class HabitipySensorEntity(StrEnum):
         key=HabitipySensorEntity.HABITS,
         translation_key=HabitipySensorEntity.HABITS,
         value_fn=lambda tasks: [r for r in tasks if r.get("type") == "habit"],
+        entity_registry_enabled_default=False,
```

Comment:
Even better, do not add these entities if they are disabled (or did not exist already)

Reply:
I could do that, but why?

Reply:
Because you intend to remove them in the future or is there something I don't understand with the PR?

So why not remove them now for users who are clearly not using them so they don't start using them now and then they are gone in near time.